code/game/objects/items/weapons/clown_items.dm

```diff
@@ -59,6 +60,11 @@
 	if(honk())
 		add_fingerprint(user)
 
+/obj/item/weapon/bikehorn/Crossed(var/mob/living/AM)
+	if (world.time > next_honk)
+		honk()
+		next_honk = world.time + 2 SECONDS
```

Comment:
I'd recommend making the delay a variable

Reply:
Done, used honk_delay